src/StackHCI/custom/stackhci.ps1

```diff
@@ -5563,7 +5563,11 @@ function Install-AzStackHCIRemoteSupport{
     param()
 
     Setup-Logging -LogFilePrefix "AzStackHCIRemoteSupportInstall" -DebugEnabled ($DebugPreference -ne "SilentlyContinue")
-    if(Assert-IsObservabilityStackPresent){
+
+    $agentInstallType = (Get-ItemProperty -Path "HKLM:\SYSTEM\Software\Microsoft\AzureStack\Observability\RemoteSupport" -ErrorAction SilentlyContinue).InstallType
+    $observabilityStackPresent = Assert-IsObservabilityStackPresent
+
+    if($observabilityStackPresent -or ($agentInstallType -eq "ArcExtension")){
         Write-InfoLog("Install-AzStackHCIRemoteSupport is not available.")
```

Comment:
Write-InfoLog("Install-AzStackHCIRemoteSupport is not available. ObsStackPresent: <True or False>, AgentInstallType: <"ArcExtension" or ""> ")